core/services/pipeline/runner.go

```diff
@@ -264,7 +264,8 @@ func (r *runner) run(
 	vars Vars,
 	l logger.Logger,
 ) (TaskRunResults, error) {
-	l.Debugw("Initiating tasks for pipeline run of spec", "job ID", run.PipelineSpec.JobID, "job name", run.PipelineSpec.JobName)
+	l.With("jobID", run.PipelineSpec.JobID, "jobName", run.PipelineSpec.JobName).
+		Debugw("Initiating tasks for pipeline run of spec")
```

Comment:
Can drop the `w` suffix?
```suggestion
		Debug("Initiating tasks for pipeline run of spec")
```
FWIW, the `w` suffix variants are more efficient.